npm-shrinkwrap.json

```diff
@@ -4159,7 +3876,8 @@
     "type-detect": {
       "version": "1.0.0",
       "from": "type-detect@>=1.0.0 <2.0.0",
-      "resolved": "https://registry.npmjs.org/type-detect/-/type-detect-1.0.0.tgz"
+      "resolved": "https://registry.npmjs.org/type-detect/-/type-detect-1.0.0.tgz",
+      "dev": true
```

Comment:
not sure - there are other dependencies in the tree that aren't modified that have dev true. Were you having issues building/running the gui with these changes?